lib/Sketchfab.js

```diff
@@ -32,67 +33,60 @@ function Sketchfab(auth, config) {
 
 }
 
-Sketchfab.prototype.upload = function(options, callback) {
+Sketchfab.prototype.upload = function ( options, callback ) {
 
-    if (options.file) {
-        fs.stat(options.file, function onFileCheck(err, stats) {
-            if (stats.isFile()) {
-                var task = new Task(this._auth, this._config, options, stats);
-                callback(null, task);
+    if ( options.file ) {
+        fs.stat( options.file, function onFileCheck( err, stats ) {
+            if ( stats.isFile() ) {
+                var task = new Task( this._auth, this._config, options, stats );
+                callback( null, task );
             } else {
-                callback('File not found ' + options.file, null);
+                callback( 'File not found ' + options.file, null );
             }
-        }.bind(this));
+        }.bind( this ) );
     } else {
-        callback('No file specified', null);
+        callback( 'No file specified', null );
     }
 
 };
 
-Sketchfab.prototype._get = function(endpoint, callback) {
+Sketchfab.prototype._get = function ( endpoint, callback ) {
     var req;
 
-    if (this._auth.type === 'token') {
+    if ( this._auth.type === 'token' ) {
 
-        req = request({
+        req = request( {
             method: 'GET',
-            uri: this._config['BASE_API_URL'] + endpoint,
+            uri: this._config[ 'BASE_API_URL' ] + endpoint,
             qs: {
                 token: this._auth.token
             }
-        }, callback);
+        }, callback );
 
-    } else if (this._auth.type === 'oauth2') {
+    } else if ( this._auth.type === 'oauth2' ) {
 
-        req = request({
+        req = request( {
             method: 'GET',
-            uri: this._config['BASE_API_URL']  + endpoint,
+            uri: this._config[ 'BASE_API_URL' ] + endpoint,
             headers: {
                 'Authorization': 'Bearer ' + this._auth.token
             }
-        }, callback);
+        }, callback );
 
     }
 
 };
 
-Sketchfab.prototype.me = function(callback) {
+Sketchfab.prototype.me = function ( callback ) {
 
-    this._get('/v2/users/me', function(err, response, body) {
-        var data;
-
-        if (response.statusCode === 200) {
-            data = JSON.parse(body);
-            callback(err, data);
+    this._get( CONFIG.ME_ENDPOINT, function ( err, response, body ) {
+        var parsedBody = JSON.parse( body );
```

Comment:
would add try/catch on JSON.parse to handle people feeding bad url/api calls, and thus getting http error code instead of json